subsys/bluetooth/mesh/Kconfig.models

```diff
@@ -12,6 +12,25 @@ config BT_MESH_NRF_MODELS
 	  Common mesh model support modules, required by all Nordic BT mesh
 	  models.
 
+menu "Configuration of the waiting time for acknowledged messages"
+
+comment "Mesh Profile recommends the waiting time a minimum of 30 seconds"
```

Comment:
There's no way to know this in advance, but the `comment` mechanism is primarily used as separators or headings for the config menu, and not as actual comments. I suggest to put this line in the help text. 
It looks a bit odd to not use the recommended value by default, but neither the timeout base nor the hop time is really the wait time, so it doesn't matter that much.

Reply:
done